.github/workflows/benchmark.yml

```diff
@@ -1,6 +1,6 @@
 name: Benchmark GTSFM on select datasets using SIFT and Deep front-ends
 
-on: [pull_request, workflow_dispatch]
```

Comment:
why are we removing this?

Reply:
Just for testing to avoid spawning a bunch of actions everytime I commit, will add back before merge

Reply:
approval pending this important change.